BTVN10/main.js

```diff
@@ -0,0 +1,167 @@
+//b1:in bang nhan
+const getMultiply = (number) => {
+    console.log('In bang nhan:')
+    for(let i=1; i<=10; i++){
+        console.log(number, 'x', i, '=', number*i)
```

Comment:
Dùng string template cho đỡ rối:

```suggestion
        console.log(`${number} * ${i} = ${number * i}`)
```